pages/reviewer/index.tsx

```diff
@@ -37,6 +37,54 @@ function ReviewerPortal() {
           </div>
         </div>
       </div>
+      <div className="bg-gray-200 rounded-xl py-5 px-3 sm:py-5 sm:px-10 lg:px-32 my-8">
+        <p className="mx-auto font-black text-md text-red-850 text-center mt-1">
+          Sponsor a student
+        </p>
+        <div className="flex justify-center space-x-5">
+          <div className="">
+            <a
+              href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=MYCSXB9B4ENP6"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <button className="px-10 py-1 block mx-auto bg-red-850 text-sm font-black text-white text-center rounded-3xl mt-2">
+                PayPal/Credit Card
+              </button>
+            </a>
+          </div>
+          <div>
+            <a
+              href="https://account.venmo.com/u/SSGSA"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <button className="px-10 py-1 block mx-auto bg-red-850 text-sm font-black text-white text-center rounded-3xl mt-2">
+                Venmo
+              </button>
+            </a>
+          </div>
+          <div>
+            <button className="px-10 py-1 block mx-auto bg-red-850 text-sm font-black text-white text-center rounded-3xl mt-2">
+              <details open>
+                <summary>Zelle</summary>
+                <p>
+                  Username: SSGSA
+                  <br />
+                  Phone number: 0000000000
```

Comment:
Would we need to make our phone number public?

Reply:
I don't think a phone number is required here. Moreover, this button is not redirecting anywhere, it is just showing the Zelle account information. So if the Zelle payment requires the phone number then it should be kept here.

Reply:
I don't think a phone number is required here. Moreover, this button is not redirecting anywhere, it is just showing the Zelle account information. So if the Zelle payment requires the phone number then it should be kept here.